daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -62,6 +62,12 @@ A component may skip the Beta stage and conformance test requirement per the dis
 - The component has been available as Alpha or Beta for at least 1 minor version release of Dapr runtime prior
 - A maintainer will address component security, core functionality and test issues according to the Dapr support policy and issue a patch release that includes the patched stable component
 
+{{% alert title="Note" color="primary" %}}
+While Dapr claims a component is stable, it is based on Dapr's tests and not a guarantee of support by any specific vendor. 
+
+Dapr tests guarantee the stability of a component independent of a third party vendor's stability status. This is because the meaning of stable (or alpha and beta) can vary for each vendor. 
```

Comment:
```suggestion
Dapr component tests guarantee the stability of a component independent of a third party vendor's declared stability status for any SDKs used. This is because the meaning of stable (for example alpha, beta, stable) can vary for each vendor. 
```

Reply:
Looks good.